app/views/admin/move_advisers/choose_subsidiary.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_for(@form, url: confirm_admin_move_advisers_path, method: :get) do |f| %>
+  <h1>Choose subsidiary</h1>
+  <% if @form.errors.any? %>
+    <ul class="t-errors bg-danger">
+      <% @form.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
```

Comment:
It depends on their styles, probably best to leave the conditional to avoid stray elements